### PR TITLE
Loan: Make structured answer

### PR DESCRIPTION
**Pull request by kirkins**
## Description of new Instant Answer, or changes

- Convert to structured answer.
- Replace tabs with spaces.

## Related Issues and Discussions

#1163

## Testing & Review
To be completed by Community Leader (or DDG Staff) when reviewing Pull Request

**Pull Request**
- [ ] Title follows correct format (Specifies Instant Answer + Purpose)
- [ ] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Instant Answer Page** (for new Instant Answers)
- [ ] Instant Answer page is correctly filled out and contains:
    - The **Title** is appropriately named and formatted
    - The **IA topics** are present and appropriate
    - The **Description** is clear and coherent
    - **Source Name** exists if applicable
    - All **Example Queries** trigger on [Beta](https://beta.duckduckgo.com/)

**Code**
- [ ] Adheres to the [DuckDuckGo Style Guide](https://docs.duckduckhack.com/resources/code-style-guide.html)
- [ ] Behaviour is appropriately tested. If improvement, tests are adequately extended.
- [ ] There is no unnecessary files in place (such as editor config files)
- [ ] There is no API keys / secrets present
- [ ] Tests are passing (run `$ duckpan test <goodie_id>`)
    - Tester should report any failures

**Ready to merge?**

- [ ] Has this IA been deployed to and tested on [beta.duckduckgo.com](https://beta.duckduckgo.com/)?
- [ ] For larger commits, has this been approved be more than one community member?
- [ ] The number of reviews is appropriate for this type of PR
- [ ] The commit message is clear, coherent and fitting

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/programming-mission/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/Loan

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/DDG/Goodie/Loan.pm [hunk 1]
```diff
@@ -139,9 +139,19 @@ handle remainder => sub {
 	my $monthly_payment = loan_monthly_payment($loan_amount, $rate / 12 * .01, $years * 12);
 	my $total_interest = ($monthly_payment * 12 * $years) - $loan_amount;
 
-	return "Monthly Payment is " . currency_format($currency_code, $monthly_payment, FMT_SYMBOL) .
+	my $string_answer = "Monthly Payment is " . currency_format($currency_code, $monthly_payment, FMT_SYMBOL) .
 	    " for $years years. Total interest paid is " .
 	    currency_format($currency_code, $total_interest, FMT_SYMBOL);
+        my $title = currency_format($currency_code, $monthly_payment, FMT_SYMBOL) . " for $years years";
```
Comment (mintsoft) [inline lib/DDG/Goodie/Loan.pm]: I think we're probably better off with "$395.31 per month for 5 years" as the title and "Total interest paid: $1,718.67" as the subtitle

**Comment by mintsoft** [comment]
I think it can be a little clearer with less repetition and being more precise with the monthly payment

screenshot_2017-07-18_00-27-05

File: lib/DDG/Goodie/Loan.pm (hunk 1 above)
Comment (kirkins) [inline lib/DDG/Goodie/Loan.pm]: Good idea I like that.

File: lib/DDG/Goodie/Loan.pm [hunk 2]
```diff
@@ -51,7 +51,7 @@ sub normalize_formatted_currency_string {
 
     $str =~ s/\Q$thousands_separator//g;
     if ($decimal_separator ne ".") {
-	$str =~ s/\Q$decimal_separator/\./g;
+    $str =~ s/\Q$decimal_separator/\./g;
```
Comment (kirkins) [inline lib/DDG/Goodie/Loan.pm]: Looks like the format got messed up when I replaced tabs with spaces.

**Comment by pjhampton** [comment]
Travis is now fixed. Please rebase and push your work to get the updated build information 👍

File: .travis.yml [hunk 3]
```diff
@@ -1,4 +1,5 @@
 ---
+dist: precise
```
Comment (mintsoft) [inline .travis.yml]: I don't think you've rebased correctly here, this shouldn't show up in the diff

**Comment by mintsoft** [comment]
LGTM Thanks @kirkins

**Pull request merged**